src/client/styles/card.scss

```diff
@@ -0,0 +1,51 @@
+/**
+ *
+ *  Online store PWA sample.
+ *  Copyright 2017 Google Inc. All rights reserved.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License
+ *
+ */
+ .card {
+   box-shadow: 0 4px 5px 0 rgba(0,0,0,.14), 0 1px 10px 0 rgba(0,0,0,.12), 0 2px 4px -1px rgba(0,0,0,.2);
```

Comment:
Nit: let's add CSS rules in alphabetical order. Best practice and avoids linter complaints.

Reply:
I'm Chaotic Evil. You?

![lawful-good-ordering-css-properties-by-logical-grouping-neutral-good-28163412](https://user-images.githubusercontent.com/1914261/37519725-8da588a8-291a-11e8-87e7-b7f418c1af9a.png)